test_script.py

```diff
@@ -0,0 +1,5 @@
+import pandas
```

Comment:
Instead, add tests in `pandas/tests/indexes/datetimes/test_misc.py` near the other `day_name` and `month_name` tests

Reply:
My bad! I didn't mean to stage this file. I'll remove this